### PR TITLE
feat: Cart management added

### PR DESCRIPTION
**Pull request by Manishsharma203**
For review refer link below:
`https://7nnb2y.csb.app/pages/cart.html`

### CODE REVIEW

**Review by krgarima — COMMENTED** [review]
Some suggestions on styling:
1. Give some margin between products in Cart. They are too close.
2. 'Proceed to Checkout' button is too wide. You can reduce its width.
3. The photo in second card is sometimes loading and not loading. The problem is with 'unsplash' I guess. Storing images locally can help.
m

**Comment by Manishsharma203** [comment]
> Some suggestions on styling:
> 
>     1. Give some margin between products in Cart. They are too close.
> 
>     2. 'Proceed to Checkout' button is too wide. You can reduce its width.
> 
>     3. The photo in second card is sometimes loading and not loading. The problem is with 'unsplash' I guess. Storing images locally can help.
>        m

1. Noted
2. That was  intentional but will reduce it a bit
3. Using locally only. Will look once more

File: index.html [hunk 1]
```diff
@@ -1,11 +1,61 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="style.css">
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer">
+      <div class="navbar bg-primary p-2" style="z-index: 100">
```
Comment (arulk8) [inline index.html:15]: pls use class for z-index. inline style should be avoided

Comment (arulk8) [reply index.html:15]: Good to merge.

Comment (Manishsharma203) [reply index.html:15]: Thanks for pointing it out. Forgot about it

**Pull request merged**